boards/shields/max3421e/shield.yml

```diff
@@ -0,0 +1,4 @@
+shields:
+  - name: sparkfun_max3421e
+    full_name: SparkFun USB-C Host Shield
```

Comment:
`    full_name: SparkFun USB Host Shield`

Reply:
This is the actual name of the product https://www.sparkfun.com/sparkfun-usb-c-host-shield.html, I guess due to having a USB-C _connector_. So, I would like to suggest sticking with it. But thanks for paying attention as I may definitely have gotten things wrong as part of the "conversion" process!

Reply:
When I added it, it definitely was not, and the version I have on my desk has a Type A connector. It looks like the USB-C version is new, so I suggest keeping the name "SparkFun USB Host Shield".

Reply:
ah, gotcha. Fixed.
FWIW, from a quick look, the new-ish USB-C version looks compatible with the old "USB Host Shield" https://github.com/sparkfun/SparkFun_USB-C_Host_Shield